### ISSUE TITLE
[Bug] Fix No Alt Texts

### ISSUE THREAD

**Issue by Annahbellah** [creativecommons/vocabulary#85]
## Description
On the homepage of the Vocabulary website,  there is no alt text provided to describe the missing images for users with disabilities, such as individuals who are visually impaired.

## Expectation
All images should include descriptive alt text to improve accessibility.

## Environment

 - Device: ( laptop)
 - OS: (macOS Ventura 13.2.1)
 - Browser: (Chrome)
 
## Additional context
The issue will be  resolved by adding appropriate alt text for accessibility.

## Resolution
 I would be interested in resolving this bug.

**Comment by possumbilities** [comment]
Hi @Annahbellah Can you please be more specific in what you mean by "Homepage of the Vocabulary website". The docs site for it does not have any images, and if you are referring to a context page within the system, like maybe the `home-index.html`? Which images are you referring to specifically?

Moving to `status: ticket work required`.

**Comment by Annahbellah** [comment]
Thank you @possumbilities the page I'm referring to is the `home-index.html` in the context page within the system. The images are in the following sections :

- The images in the list inside the `<article class="case-studies">` section.
- All the images in the `<article class="stories authored-posts highlight">` section.
-  All the images under the` <a class="more" href="#">more posts</a>` section.

**Comment by MahtabShah** [comment]
i can solve , can u assign me.........?
